### PR TITLE
Format code consistently using pre-commit

### PR DESCRIPTION
**Pull request by yoomlam**
## What was the problem?

Files were inconsistently formatted in terms of end-of-file and trailing-whitespace.
Due to the new `pre-commit` hook (#659), folks will have to address them on the next modification of a non-conforming file.

## How does this fix it?

Fix most of the pre-commit alerts in this PR to reduce the burden on developers.

## Tip for reviewing this PR
Enable "Hide whitespace" when reviewing to see that only formatting has been changed for all files except for file `/build.gradle`:
image

## How to test this PR
Manually test VRO to ensure functionality has NOT changed.

To see other fixes that can be done, run `pre-commit run --all-files`. Only Python files remain to be updated.

### CODE REVIEW

File: .github/workflows/publish-dev.yml [hunk 1]
```diff
@@ -106,4 +106,3 @@ jobs:
           GITHUB_USERNAME: ${{ secrets.USERNAME }}
         run: |
           ./deploy-app.sh dev ${COMMIT_SHA:0:7}
-
```
Comment (yoomlam) [inline .github/workflows/publish-dev.yml:109]: `end-of-file-fixer`: Makes sure files end in a newline and only a newline.

File: service-python/tests/assessclaim/dc7101v2/main_hypertension_test.py [hunk 2]
```diff
@@ -55,8 +55,7 @@
                     },
                     "dateOfClaim": "2021-11-09",
                     "disabilityActionType": "NEW"
-                }
-                ,
+                },
```
Comment (yoomlam) [inline service-python/tests/assessclaim/dc7101v2/main_hypertension_test.py:58]: `flake8` warned about this

File: svc-lighthouse-api/src/test/resources/testkey.pem [hunk 3]
```diff
@@ -25,4 +25,3 @@ CShLEX8CgYAI4ZboDU6SnR7jUGxDlVQMQz6+et/Xpc/bt7z6FcRDExbpqkN1qPze
 lkqFEivH0ol5gqIgM09lhuKkZBkALoOHDVyD5c0c9hzkEztOdhIIfYYifI/+GK/y
 wNvBzQXmbHJ+vpLcOAluJ7X2qa/wNuNUtKKgiVgj0V+GMo0dhtT+PQ==
 -----END RSA PRIVATE KEY-----
-
```
Comment (yoomlam) [inline svc-lighthouse-api/src/test/resources/testkey.pem:28]: I assume this is acceptable.

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -27,6 +27,7 @@ repos:
         exclude: ^.*helmchart.*$
       - id: destroyed-symlinks
       - id: detect-private-key
+        exclude: ^svc-lighthouse-api/.*/testkey\.pem$
```
Comment (yoomlam) [inline .pre-commit-config.yaml:30]: This private key is only for testing.

**Review by au2 — APPROVED** [review]
All tests pass including end 2 end.

**Review by anthonyvelazquez — APPROVED** [review]

**Pull request merged**